test/transmuter/TransmuterUpdateFacetsTest.t.sol

```diff
@@ -0,0 +1,353 @@
+    bytes public oracleConfigDataBC3M =
+        hex"00000000000000000000000000000000000000000000000000000000000000a000000000000000000000000000000000000000000000000000000000000000e000000000000000000000000000000000000000000000000000000000000001200000000000000000000000000000000000000000000000000000000000000160000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000010000000000000000000000006e27a25999b3c665e44d903b2139f5a4be2b6c260000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000003f4800000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000000100000000000000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000008";
+    bytes public oracleConfigDataBERNX =
+        hex"00000000000000000000000000000000000000000000000000000000000000a000000000000000000000000000000000000000000000000000000000000000e00000000000000000000000000000000000000000000000000000000000000120000000000000000000000000000000000000000000000000000000000000016000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000001000000000000000000000000475855dae09af1e3f2d380d766b9e630926ad3ce0000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000003f4800000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000000100000000000000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000008";
+    //
+
+    function setUp() public override(Utils, BaseTest) {
+        BaseTest.setUp();
+
+        // special case as we rely on the fork state
+        vm.selectFork(forkIdentifier[CHAIN_FORK]);
+        // vm.selectFork(forkIdentifier[chainId]);
+
+        vm.pauseGasMetering();
+
+        // As there are calls to price feeds and there are delays to be respected we need to mock calls
+        // to escape from `InvalidChainlinkRate()` error
+        vm.mockCall(
+            address(0x6E27A25999B3C665E44D903B2139F5a4Be2B6C26),
+            abi.encodeWithSelector(AggregatorV3Interface.latestRoundData.selector),
+            abi.encode(uint80(0), int256(11949000000), uint256(block.timestamp), uint256(block.timestamp), uint80(0))
+        );
+
+        uint256 chainId = json.readUint("$.chainId");
+
+        address gnosisSafe = _chainToContract(chainId, ContractType.GovernorMultisig);
+
+        transmuter = ITransmuter(_chainToContract(chainId, ContractType.TransmuterAgEUR));
+
+        address to = json.readAddress("$.to");
+        // uint256 value = json.readUint("$.value");
```

Comment:
it is not real usefull but it should be a best practice to keep the value and send it to the gnosisSafe.call() as we never know if we wants to change the script someday